src/components/FrontPageContentTile.svelte

```diff
@@ -5,6 +5,6 @@
  -->
 
 <div
-  class="flex flex-col flex-wrap items-center border-gray-200 border border-solid bg-white rounded-3xl p-6 shadow-xl max-w-88">
+  class="flex flex-col flex-wrap items-center border-gray-200 border border-solid bg-white rounded-3xl p-6 shadow-md max-w-88">
```

Comment:
Reducing card box shadow size. 

Windi css only conveniently supports adjusting of box shadow size and colour:
https://windicss.org/utilities/effects/box-shadow.html#box-shadow-size